pkg/sensors/exec/exec_test.go

```diff
@@ -452,7 +452,7 @@ func TestDocker(t *testing.T) {
 	observer.LoopEvents(ctx, t, &doneWG, &readyWG, obs)
 
 	readyWG.Wait()
-	serverDockerID := observer.DockerRun(t, "--name", "fgs-test-server", "--entrypoint", "nc", "quay.io/cilium/alpine-curl:1.0", "-nvlp", "8081", "-s", "0.0.0.0")
+	serverDockerID := observer.DockerRun(t, "--name", "fgs-test-server", "--entrypoint", "nc", "quay.io/cilium/alpine-curl:v1.6.0", "-nvlp", "8081", "-s", "0.0.0.0")
```

Comment:
Just a question: is this a multi-arch image?

Reply:
Yep that's exactly the reason of the patch, see commit message here https://github.com/cilium/tetragon/pull/734/commits/6cd5873885f24b8ca396ea8a380f4549d4fb933f